decentraland/AppNavigator.js

```diff
@@ -16,34 +16,64 @@ import EthereumQuestionScreen from "./screens/EthereumQuestionScreen";
 import EthereumSignUpScreen from "./screens/EthereumSignUpScreen";
 import EthereumSignInScreen from "./screens/EthereumSignInScreen";
 import MyAssetsScreen from "./screens/MyAssetsScreen";
+
 import Colors from "@constants/Colors.js";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+
+const defaultNavigationOptions = {
+  headerStyle: {
+    backgroundColor: Colors.headerBackgroundColor,
+    height: responsiveHeight(7),
```

Comment:
Note that we'll want to see how this looks on an iPhone X where there's the weird tab at the top (my personal device in an iPhone X)

Reply:
Ok, here the headers' height was decreased a little.

Reply:
I’ll resolve this thread and open an issue if anything looks off on iOS